prime-router/settings/organizations.yml

```diff
@@ -104,25 +104,24 @@
       organizationName: ignore
       topic: full-elr
       customerStatus: active
-      schemaName: metadata/fhir_transforms/senders/default-sender-transform
+      schemaName: metadata/fhir_transforms/senders/original-pipeline-transforms
```

Comment:
These changes can't/shouldn't be made in staging until deployment, but after deployment, if anybody is referencing `default-sender-transform`, then things will break until we can update settings. It can happen more seamlessly if we keep a placeholder for `default-sender-transform` to prevent breakage, deploy, update the settings, and then go back and delete the `default-sender-transform` for the next deployment. That might be overkill, but you should probably check if we have any orgs referencing the default, and then we can decide how to proceed.

Reply:
No, they have not been made in staging. They will need to be made there right before merging.

Reply:
`you should probably check if we have any orgs referencing the default, and then we can decide how to proceed.`

I like this idea. Can we compile this list for both staging and production environments and put it in the ticket description? Also, can we write a plan in the ticket description of how we are going to proceed once this PR is merged. Once it is merged, we can update staging and run smokeTests, right?. Once we deploy, we can update production.

Reply:
Compiled and added to the ticket. Jimmy actually put a plan in the comments in the ticket already, so I think we are good to go there as well.